package.json

```diff
@@ -47,8 +47,12 @@
 		"@types/jsonwebtoken": "8.5.9",
 		"@types/passport": "1.0.11",
 		"@types/passport-google-oauth20": "2.0.11",
+		"@typescript-eslint/eslint-plugin": "^5.38.1",
+		"@typescript-eslint/parser": "^5.38.1",
 		"add": "2.0.6",
 		"all-contributors-cli": "6.22.0",
+		"eslint": "^8.24.0",
+		"eslint-plugin-react": "^7.31.8",
```

Comment:
remove react plugin because we don't need it in backend

Reply:
ok